presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -428,29 +428,27 @@ public TaskManagerConfig setInfoMaxAge(Duration infoMaxAge)
     }
 
     @Min(1)
-    @PowerOfTwo
```

Comment:
This change fully removes it from the power of two check from the config checking. I don't see an easy way to gate any kind of check based on whether native execution is disabled, but this should maybe have at least a code comment explaining the constraint.